include/openPMD/RecordComponent.hpp

```diff
@@ -387,6 +404,9 @@ class RecordComponent : public BaseRecordComponent
      */
     RecordComponent &makeEmpty(Dataset d);
 
+    void storeChunk(
+        auxiliary::WriteBuffer buffer, Datatype datatype, Offset o, Extent e);
+
```

Comment:
private, no new API

Reply:
(could nonetheless have Doxygen block for us ;-) )